src/components/avatar/__tests__/Avatar-test.js

```diff
@@ -66,18 +72,27 @@ describe('components/avatar/Avatar', () => {
             avatarUrl: 'http://foo.bar/baz123_invalid',
         };
 
-        const wrapper = shallow(<Avatar {...props} />);
-
-        wrapper.instance().onImageError();
-        expect(wrapper.state('hasImageErrored')).toEqual(true);
+        let wrapper;
+        act(() => {
```

Comment:
It seems a bit arbitrary when we're using `act` or not. Can we create a `getWrapper` helper that abstracts it away?

Reply:
yes, but i'm not sure how that would help the additional calls to `act`.

```
const wrapper = getWrapper(<Avatar {...props} />);
wrapper.find(...);
// Auto call wrapper.update after
wrapper.act({
   ....
})
```
Something like that?

Reply:
I think for now, we should just use `act` as is and wait for enzyme to fix the update functionality.